expo-maps/src/Maps.tsx

```diff
@@ -17,13 +17,73 @@ const defaultNativeExpoMapViewProps: DefaultNativeExpoMapViewProps = {
 };
 
 export class ExpoMap extends React.Component<ExpoMapViewProps> {
+  private mapChildren() {
+    const childrenArray = React.Children.map(this.props.children, (child) => {
+      if (isNotSimpleType(child)) {
+        if (instanceOfExpoMarker(child)) {
+          return {
+            type: 'marker',
+            lat: child.props.latitude,
+            lng: child.props.longitude,
```

Comment:
I know it's just an internal format, but I would recommend using full names too 😉 Especially that records are automatically validating those values and if they are not valid, the appropriate error will be thrown and its message contains record's field name, so the user would see `lat` and `lng` there.